tensorboard/webapp/plugins/npmi/util/sort_annotations_test.ts

```diff
@@ -52,15 +52,11 @@ describe('sort annotations utils', () => {
 
   it('sorts annotations by similar embeddings', () => {
     const annotationData: AnnotationDataListing = buildSampleAnnotationData();
+    const embeddingData = buildSampleEmbeddingData();
```

Comment:
Philosophy thing: As a reader of the code, I should be able to read things in the spec and I should be able to have confidence that the code is correct. With the input folded into a utility, I have to flip through files to know that the sortAnnotation is correct which is not ideal. Can you keep below?
```ts
annotation_1: [0.5],	
annotation_2: [-0.2],
annotation_3: [0.1],
```

Reply:
I don't think so. The data structure changed from EmbeddingLising to EmbeddingDataSet, which is no initializable like this.